webapp/scripts/functions.js

```diff
@@ -269,16 +314,31 @@ function parseDescription(raw) {
  * @param {*} mapset mapset name
  * @param {*} table table name
  */
-function addBounds(desc, mapset, table) {
-  desc.columnObj.rows.forEach(row => {
-    if (['DOUBLE PRECISION', 'INTEGER'].indexOf(row.type) > -1) {
-      const [min, max] = module.exports.getUnivarBounds(mapset, table, row.column)
-      row.min = min
-      row.max = max
+function setBounds(desc, mapset, table) {
+  let i = 0
+  while (i < desc.columnObj.rows.length) {
+    const bounds = module.exports.getUnivarBounds(mapset, table, desc.columnObj.rows[i].column)
+    // add bounds when they exist, remove element when they don't
+    if (bounds.length) {
+      desc.columnObj.rows[i].min = bounds[0]
+      desc.columnObj.rows[i].max = bounds[1]
+      i++
+    } else {
+      desc.columnObj.rows.splice(i, 1)
```

Comment:
This loop is tricky. It would probably look nicer if you used array functions and separated the two concerns, "add bounds when they exist, remove element when they don't"

Reply:
This function will be altered when the attributes expand to include CHARACTER-types